cypress/e2e/tests/pages/explorer/dashboard/cluster-dashboard.spec.ts

```diff
@@ -257,19 +257,9 @@ describe('Cluster Dashboard', { testIsolation: 'off', tags: ['@explorer', '@admi
   });
 
   it('can view events table empty if no events', { tags: ['@vai', '@adminUser'] }, () => {
-    cy.visit(clusterDashboard.urlPath(), {
-      onBeforeLoad(win) {
-        cy.stub(win.console, 'error').as('consoleError');
-        cy.stub(win.console, 'warn').as('consoleWarn');
-      },
-    });
-
     eventsNoDataset();
     clusterDashboard.goTo();
 
-    cy.get('@consoleError').should('not.be.called'); // See error lot
-    cy.get('@consoleWarn').should('not.be.called'); // See warning log (there will be some....)
```

Comment:
Can this go back in? Are we avoiding warnings and errors or is it just not necessary anymore?

Reply:
This was erroneously added in the previous PR https://github.com/rancher/dashboard/pull/11663/files#diff-2e0d69f6ba95f5b63f312a8b8d2001db4b2a63ad3499de89103564b933df32bd